views/layout.slim

```diff
@@ -3,6 +3,7 @@ html
     head
         title My Expenditure App
         link rel="stylesheet" type="text/css" href="css/style.css"
+        script src = "javascript/main.js"
```

Comment:
It is usually advisable to have your js file load last so it does not block the rest of the page from rendering.

Reply:
should i put it on the last line?